src/js/utils/DOM.js

```diff
@@ -157,7 +157,7 @@ export const makeNodeFocusable = (node) => {
   }
 };
 
-const autoFocusingTags = /(a|area|input|select|textarea|button|iframe)$/;
+const autoFocusingTags = /(a|area|input|select|textarea|button|iframe)(?![\w-])$/;
```

Comment:
This RE still matches tags like `meta`. What is your addition trying to do? I would think a fix would be anchoring the beginning. For example:
```jsx
const autoFocusingTags = /^(a|area|input|select|textarea|button|iframe)$/;
```